src/main/kotlin/com/seokjoo/todo/domain/service/auth/TodoAuthService.kt

```diff
@@ -65,6 +65,7 @@ class TodoAuthService(
         return jwtProvider.generateToken(userId, JwtTokenType.REFRESH)
     }
 
+    @Transactional
     fun findUser(accessToken: String): User {
```

Comment:
여기서 필요해요?

Reply:
ㄴㅔ넹 AuthRepository에서 유저 정보 가져오니 필요하지 않나요? 근데 readOnly는 필요하겠어요
```suggestion
    @Transactional(readOnly = true)
    fun findUser(accessToken: String): User {
```

Reply:
네넹 `readOnly` 붙이면 더 좋을 듯!
근데 조회 기능만 있어서 트랜잭션 범위로 꼭 묶을 필요는 없어 보여 물어봤어요